cpp/examples/glct_secir.cpp

```diff
@@ -0,0 +1,171 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "glct_secir/model.h"
+#include "glct_secir/parameters.h"
+#include "memilio/config.h"
+#include "memilio/utils/time_series.h"
+#include "memilio/epidemiology/uncertain_matrix.h"
+#include "memilio/math/eigen.h"
+#include "memilio/utils/logging.h"
+#include "memilio/compartments/simulation.h"
+#include "memilio/data/analyze_result.h"
+
+#include <vector>
+
+int main()
+{
+    // Simple example to demonstrate how to run a simulation using an GLCT SECIR model.
+    // Parameters, initial values and the number of subcompartments are not meant to represent a realistic scenario.
+    constexpr size_t NumExposed = 2, NumInfectedNoSymptoms = 6, NumInfectedSymptoms = 2, NumInfectedSevere = 2,
+                     NumInfectedCritical = 10;
+    using Model    = mio::glsecir::Model<NumExposed, NumInfectedNoSymptoms, NumInfectedSymptoms, NumInfectedSevere,
+                                      NumInfectedCritical>;
+    using LctState = Model::LctState;
+    using InfectionState = LctState::InfectionState;
```

Comment:
This is inconsistent with the LCT example, can you adapt this?

Reply:
You are referring to the number of subcompartments, dont you? Actually, this is very consistent. In the GLCT model we just have the two strains for all compartments apart from E such that we need two times the number of subcompartments.

Reply:
Actually, I meant that you are not initializing with LctState but you already mentioned below that this will be adapted in #1134.